### DISCUSSION TITLE
Transform Modal Options

### DISCUSSION THREAD

**Discussion by adi-fi-ma** [josdejong/svelte-jsoneditor#443] [Q&A]
Hi 
I would like to trigger a custom transform modal to pop whenever the transform icon is clicked (via the editor's bar, right click etc). 
Is there any option to do so?
Thanks!

**Comment by josdejong** [comment]
Yes, you can customize the menu buttons and context menu buttons using `onRenderMenu` and `onRenderContextMenu`, you can replace what happens when clicking the Transform button.

Docs: 

- https://github.com/josdejong/svelte-jsoneditor?tab=readme-ov-file#onrendermenu
- https://github.com/josdejong/svelte-jsoneditor?tab=readme-ov-file#onrendercontextmenu
